app/javascript/packs/stylesheets/questions.scss

```diff
@@ -0,0 +1,68 @@
+@import "global";
+
+.question__container {
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.question {
+  margin-top: 10px;
+  padding: 5px 20px 5px 20px;
+  border-left: 3px solid $secondary;
+  transition: transform 0.5s;
+  transition: all 0.5s;
+}
+
+@keyframes slide {
+  from {
+  }
+
```

Comment:
interesting...how does this work? Lol

Reply:
I actually don't use this keyframe animation anymore so I'll get rid of this. I was basically setting the `from` property elsewhere in inline styles for the element.